llvm/tools/llvm-cxxfilt/llvm-cxxfilt.cpp

```diff
@@ -100,7 +99,7 @@ static std::string demangle(const std::string &Mangled, StringRef Delimiters) {
   }
 
   Result =
-      Undecorated ? Prefix + optionalQuote(Undecorated, Delimiters) : Mangled;
+      Undecorated ? optionalQuote(Prefix + Undecorated, Delimiters) : Mangled;
```

Comment:
Maybe you misunderstood something I said, but I actually prefer just the `Undecorated` part being quoted. E.g. 'import thunk for "std::future\<void\>"'. However, I'm not that bothered either way, so am happy if you'd prefer to leave it as-is.

Reply:
This was what I had first but actually to generate valid LLVM/MLIR assembly like
```mlir
%5 = cir.call @"import thunk for std::future<void>"(%1)
```
all the content has to be quoted.